client/containers/Pub/SpubHeader/SpubHeader.tsx

```diff
@@ -31,6 +32,21 @@ const SpubHeader = () => {
 	const { selectedTab, submission } = submissionState!;
 	const headerRef = useRef<HTMLDivElement>(null);
 
+	const [validator] = useState(() => createSubmissionValidator(submission.submissionWorkflow));
```

Comment:
Is this our preference for creating one-off values (as opposed to `useMemo(() => value, [])`?

Reply:
I use them both kinda interchangeably but the nice thing about `useState` here is that the linter doesn't complain about `submission` missing from the dependencies array.